github-webhook/go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/IBM/CodeEngine/github-webhook
 
 go 1.22.0
+toolchain go1.23.7
```

Comment:
@reggeenr this PR should go in. however I dont think toolchain directive is needed. Any opinion?